code/modules/mob/living/silicon/laws.dm

```diff
@@ -5,6 +5,11 @@
 	if (!laws)
 		make_laws()
 
+/mob/living/silicon/proc/log_current_laws()
+	var/list/the_laws = laws.get_law_list(include_zeroth = TRUE)
+	var/lawtext = the_laws.Join()
```

Comment:
```suggestion
	var/lawtext = the_laws.Join(" ")
```

Reply:
Let's just join them with a space in that case. Then they look a bit nicer on a single line.